traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -335,11 +335,14 @@ func getMidServers(servers []tc.ServerNullable, tx *sqlx.Tx) ([]tc.ServerNullabl
 	edgeIDs := strings.Join(ids, ",")
 	// TODO: include secondary parent?
 	q := selectQuery() + `
-WHERE t.name = 'MID' AND s.cachegroup IN (
-SELECT cg.parent_cachegroup_id FROM cachegroup AS cg
-WHERE cg.id IN (
-SELECT s.cachegroup FROM server AS s
-WHERE s.id IN (` + edgeIDs + `)))
+WHERE s.id IN (
+	SELECT mid.id FROM server mid
+	JOIN cachegroup cg ON cg.id IN (
+		SELECT cg.parent_cachegroup_id
+		FROM server s
+		JOIN cachegroup cg ON cg.id = s.cachegroup
+		WHERE s.id IN (` + edgeIDs + `))
+	JOIN type t ON mid.type = (SELECT id FROM type WHERE name = 'MID'))
```

Comment:
I don't think you meant to change this

Reply:
See [comment](https://github.com/apache/trafficcontrol/pull/4633#issuecomment-617996513), fixed in 003af41431.

Reply:
probably the result of a bad rebase. this code was changed with #4634 so let's make sure it didn't get stomped.

Reply:
Yes the second rebase replaces 6a6cffefb7 with 003af41, so this PR no longer changes that file